frontend/src/modules/machinery/components/MicrosoftTerminology.js

```diff
@@ -0,0 +1,39 @@
+/* @flow */
+
+import React from 'react';
+import { Localized } from '@fluent/react';
+
+
+type Props = {|
+    source: {
+        type: string,
+        url: string,
+    },
+|};
+
+
+/**
+ * Show the translation source from Microsoft Terminology.
+ */
+export default function MicrosoftTerminology(props: Props) {
+    const { source } = props;
+
+    return <li>
+        <Localized
+            id= "machinery-MicrosoftTerminology--visit-microsoft"
+            attrs={{ title: true }}
+        >
+            <a
+                className="translation-source"
+                href={ source.url }
+                title="'Visit Microsoft Terminology Service API.\n'+
+            '© 2018 Microsoft Corporation. All rights reserved.'"
```

Comment:
Wrong indentation here.

Reply:
And weird quotes. Is that syntax even valid??? The double quotes `"` shouldn't be needed.